checker/checker_dartagnan.go

```diff
@@ -102,17 +102,17 @@ func (c *DartagnanChecker) Check(ctx context.Context, m DumpableModule) (cr Chec
 		return cr, err
 	}
 	sout, err := c.run(ctx, testFn)
-	if err != nil {
-		return cr, err
-	}
-
-	logger.Debug(sout)
 	if ctx.Err() == context.Canceled {
 		return cr, nil
 	}
 	if ctx.Err() == context.DeadlineExceeded {
 		return CheckResult{Status: CheckTimeout}, nil
 	}
+
+	logger.Debug("Output:", sout)
+	if err != nil {
+		return cr, err
+	}
 	if err != nil {
 		return cr, err
 	}
```

Comment:
Isn't the check actually useless since we have the same check in the following 3 lines?

It seems to me that the check at lines 105-107 is the remaining of some code that was there and got deleted.

Reply:
Good catch. thanks!